### PR TITLE
update guest_create API to support boot from volume

### PR DESCRIPTION
**Pull request by dongyanyang**
add two new parameter to guest_create:
dedicate_vdevs: used to dedicate devices like fcp;
loaddev: used to add the loaddev portname and lun statement in the guest definition

### CODE REVIEW

File: smtLayer/makeVM.py [hunk 1]
```diff
@@ -134,6 +138,20 @@ def createVM(rh):
         if reservedSize != '0M':
             dirLines.append("COMMAND DEF STOR RESERVED %s" % reservedSize)
 
+    if 'loadportname' in rh.parms:
+        wwpn = rh.parms['loadportname'].replace("0x", "")
+        dirLines.append("LOADDEV PORTname %s" % wwpn)
```
Comment (bjhuangr) [inline smtLayer/makeVM.py]: Could you please show me a sample of `loaddev portname` and `loaddev lun`?

**Comment by bjhuangr** [comment]
And don't forget to update wsgi and doc to the API

File: smtLayer/makeVM.py (hunk 1 above)
Comment (dongyanyang) [reply smtLayer/makeVM.py]: > Could you please show me a sample of `loaddev portname` and `loaddev lun`?
This is a VM I created with my updated API:
[root@dycomput ~]# smcli Image_Query_DM -T mkvmtst
USER MKVMTST LBYONLY 1024m 64G G
INCLUDE ZCCDFLT
COMMAND DEF STOR RESERVED 64512M
CPU 00 BASE
IPL 5C71
LOADDEV PORTname 5005076802400c1b
LOADDEV LUN 0000000000000000
LOGONBY IAASADM
MACHINE ESA 32
DEDICATE 5C71 5C71
DEDICATE 5D71 5D71

**Comment by dongyanyang** [comment]
> And don't forget to update wsgi and doc to the API

Yes, Done. Thanks for the reminder.

**Comment by bjhuangr** [comment]
LGTM

**Pull request merged**